Bookish/Views/Catalog/Edit.cshtml.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Bookish.Views.Catalog;
 
-public class BookEntry : PageModel
+public class Edit : PageModel
```

Comment:
From this class name, it's not clear to me what this class is for

Reply:
I'm not sure why that class is there to begin with, nested beneath the cshtml. I think it was autogenerated when I made Edit.cshtml. What is this meant to be used for? It must have assumed I wanted it.